postgresql.md

```diff
@@ -0,0 +1,47 @@
+# PostgreSQL
+
+## Intro
+
+PostgreSQL is a powerful, open source object-relational database system.
+
+For information on how to install and get started with PostgreSQL take a look at their official [documentation](https://www.postgresql.org/docs/13/index.html) page.
+
+## Tips
+
+After you are fully setup and running take a look at this section as it might help you. Here we display tips and processes that will help you develop better with PostgreSQL.
+
+**Database dumping** 
+
+Great for testing during development at your local machine. Here is one way of doing:
+
+**Step 1**
+
+Open your terminal and go to your Cambiatus workspace and create a folder to dump database files (folder name example: cambiatus_dbs)
+``` 
+mkdir <foldername>
+```
+*Note: The folder name  above is an example, your path may be different*
+
+Open your **psql terminal** and go to the created folder path, for instance:
+``` 
+cd workspace/cambiatus/cambiatus_dbs
+```
+*Note: The command above is an example, your path may be different*
+
+**Step 2**
+
+To dump a database called mydb into a SQL-script file:
+```
+$ pg_dump mydb > db.sql
+```
```

Comment:
I think that new developers may need more real-world instructions like connecting to the SSH server and making the dump via SSH. Here are the steps I usually use for myself. @lucca65 @victorgdev maybe we can add something like this for the dump/undump instructions?

You can connect with SSH to the remote server, do the `pg_dump` call and send the output back to `stdout` of a local machine. Here's an example, replace the names in `UPPER_CASE` with the real ones:

```sh
ssh SSH_USER@SSH_SERVER "pg_dump --exclude-table-data=_block_number_txid -d REMOTE_DB_NAME -h localhost -U REMOTE_DB_USER" > DUMP_NAME.sql 
```

We excluding the `_block_number_txid` table because it's big and we usually don't need it on the local dev machine.

It will ask for the password for `REMOTE_DB_USER`.

Undump `DUMP_NAME.sql` to your local DB:

```
psql LOCAL_DB_NAME < DUMP_NAME.sql
```

Reply:
hmm, this surely looks better! Sorry folks, I really didn't payed much attention at this part to be honest, I feel like those types of hacks should be actually be changed to proper database mock data, that can be achieved by using mock data and good tests.

I know that they are useful, but I see them as transitory, adding docs about it just feels wrong somehow.

Anyway, I'm ok on having more in depth instructions on how to hack our way into having valid data to work with